aggregator_core/src/datastore/tests.rs

```diff
@@ -3433,10 +3433,6 @@ trait TestQueryTypeExt: CollectableQueryType {
 
     fn batch_identifier_for_client_timestamps(client_timestamps: &[Time]) -> Self::BatchIdentifier;
 
-    fn shortened_batch_identifier(
-        batch_identifier: &Self::BatchIdentifier,
-    ) -> Self::BatchIdentifier;
```

Comment:
I'm assuming this was never used?

Reply:
Correct, and it appears catching unused associated trait methods is a new feature